### PR TITLE
Security scans: VSC triggers to calculate codeScanProjectBytes only for Project scans

### PR DESCRIPTION
**Pull request by laileni-aws**
## Problem
- VSC triggers `getDirSize()` function to calculate `codeScanProjectBytes` for telemetry for both `Auto-scans` and `Project scans`.
## Solution

- Disable the `codeScanProjectBytes` calculation for `Auto-scans`: Since the `codeScanProjectBytes`
 parameter is optional for `Auto-scans` and is affecting the performance, it makes sense to disable the calculation for this scenario. This will help improve the overall performance of the Auto-scans.

- Enable the `codeScanProjectBytes` calculation for `Project scans`: For `Project scans`, you should continue to calculate the `codeScanProjectBytes` using the `getDirSize()` function, as this information is likely valuable for the telemetry.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by leigaol** [comment]
In the PR statement, please elaborate on `a bit laggy` of `getDirSize`. 

Run it in a repository of 1k files with 10 to 20 folder levels, explain the time it takes to run.

**Comment by leigaol** [comment]
Can you rename this PR ?

File: packages/core/src/codewhisperer/commands/startSecurityScan.ts [hunk 1]
```diff
@@ -255,7 +255,7 @@ export async function startSecurityScan(
         codeScanState.setToNotStarted()
         codeScanTelemetryEntry.duration = performance.now() - codeScanStartTime
         codeScanTelemetryEntry.codeScanServiceInvocationsDuration = performance.now() - serviceInvocationStartTime
-        await emitCodeScanTelemetry(codeScanTelemetryEntry)
+        await emitCodeScanTelemetry(codeScanTelemetryEntry, scope)
```
Comment (leigaol) [inline packages/core/src/codewhisperer/commands/startSecurityScan.ts:258]: Does this prevent gitDirSize invocation when it comes to auto security scan? Then how about manual project level security scan?

**Comment by manmartgarc** [comment]
Bumping, really need this 👍

File: packages/core/src/codewhisperer/commands/startSecurityScan.ts (hunk 1 above)
Comment (laileni-aws) [reply packages/core/src/codewhisperer/commands/startSecurityScan.ts:258]: Manual scan is triggered once so this should be executed only once, which is exact functionality before 4/30

**Review by justinmk3 — APPROVED** [review]
Needs a changelog item. And approval from a CW team member

**Comment by leigaol** [comment]
I can approve once you add a change log item

**Review by leigaol — APPROVED** [review]
please add one more change log item

**Pull request merged**